### DISCUSSION TITLE
Drag & drop in bookmarks explorer

### DISCUSSION THREAD

**Discussion by Yadro** [alefragnani/vscode-bookmarks#526] [Ideas]
Any news on this feature?

**Comment by Yadro** [comment]
it was mentioned here https://github.com/alefragnani/vscode-bookmarks/issues/378

**Comment by alefragnani** [comment]
Hi @Yadro ,

Not at all. In fact, there is still no API available in VS Code, so extensions can’t support drag & drop, yet.

I remember seeing some issues related to that, but no ETA that I’m aware of.
